src/compiler/compile/render_dom/wrappers/InlineComponent/index.ts

```diff
@@ -133,6 +133,7 @@ export default class InlineComponentWrapper extends Wrapper {
 		const { component } = renderer;
 
 		const name = this.var;
+		block.add_variable(name);
```

Comment:
yes, @Conduitry , I'm adding `block.add_variable(name)` here so that the later on `bind:this` will notice that this variable name has been used.

Reply:
this happens in elements as well: https://github.com/sveltejs/svelte/blob/4135ca5415dcf0003c2f14df0c1c94ee99eb7e51/src/compiler/compile/render_dom/Block.ts#L177